main.go

```diff
@@ -435,6 +436,16 @@ func main() {
 				setupLog.Error(err, "Error init CRD metrics")
 			}
 
+			if cfg.PrometheusCRAvailability() == prometheus.Available {
```

Comment:
Why is this in the block for ` os.Getenv("ENABLE_WEBHOOKS") != "false" ` The operator metrics should be enabled regardless of the webhooks.

Reply:
There is `addDependencie` where we call ` mgr.Add(`

Reply:
Oh I see now. Thanks!